charts/icm-as/tests/persistence-sites_test.yaml

```diff
@@ -113,54 +112,6 @@ tests:
             persistentVolumeClaim:
               claimName: "icm-as-cluster-sites-pvc"
 
-  - it: type=cluster in cluster-sc
```

Comment:
How is the whole storage class removal related to memory changes of JDK17?

Reply:
Ask Karsten.

Reply:
Using the new alpha release of the icm-helm-chart resulted in iste "storageClass" errors. Since the jvm changes are already in the icm-as develop branch, the here underlying helm-chart is used in our production ISTE so I had to fix the "storageClass" error immediatelly.

Reply:
I reverted the commits and created a new PR.